PyInstaller/hooks/hook-eyed3.plugins.py

```diff
@@ -0,0 +1,12 @@
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules
+from os.path import basename, splitext
+
+plugins = filter(lambda data: basename(data[0])[0] not in ('_', '.') and data[0].endswith('.py'),
+        collect_data_files('eyed3.plugins', include_py_files=True))
+
+datas = plugins
```

Comment:
Sure these should become `date` files and not hidden imports? The former will not be scanned for further dependencies.

Reply:
eyed3 searches '.py' files in the plugins directory and load them. So I have to add them to both datas and hidden imports.